mods/force-thick-frames.wh.cpp

```diff
@@ -0,0 +1,86 @@
+// ==WindhawkMod==
+// @id              force-thick-frames
+// @name            Force thick frames
+// @description     Force windows to have thick frames like in Windows Vista/7. 
+// @version         2
+// @author          Alcatel
+// @github          https://github.com/arukateru
+// @include         *
+// @exclude         notepad++.exe
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme== 
+/*  
+Force windows to have thick frames like in Windows Vista/7. This theme should already have thick borders (so like, not the default Windows 10 or 11 theme)
+![Screenshot](https://raw.githubusercontent.com/arukateru/ThickFramer/main/251890469-a04311e7-bbed-449f-9c92-79c642c36005.png) 
+*/ 
+// ==/WindhawkModReadme== 
+
+using PUNICODE_STRING = PVOID;
```

Comment:
Now the empty line before the screenshot is gone... Sorry for the ping pong, I'll try using the GitHub suggestion feature to propose a change. I should have done this in the beginning.

```suggestion
// ==WindhawkMod==
// @id              force-thick-frames
// @name            Force thick frames
// @description     Force windows to have thick frames like in Windows Vista/7
// @version         2
// @author          Alcatel
// @github          https://github.com/arukateru
// @include         *
// @exclude         notepad++.exe
// ==/WindhawkMod==

// ==WindhawkModReadme==
/*
Force windows to have thick frames like in Windows Vista/7. This theme should already have thick borders (so like, not the default Windows 10 or 11 theme).

![Screenshot](https://raw.githubusercontent.com/arukateru/ThickFramer/main/251890469-a04311e7-bbed-449f-9c92-79c642c36005.png)
*/
// ==/WindhawkModReadme==

using PUNICODE_STRING = PVOID;

```

Reply:
You can see there are other small changes, for example I removed the period from the description. You should be able to accept this suggestion with the "Commit suggestion" button if it looks good to you.